include/COpenGLStateManagerImpl.h

```diff
@@ -61,8 +61,8 @@
 #define glBindBufferRange_MACRO glBindBufferRange
 #endif // glBindBufferRange_MACRO
 
-#ifndef glGetNamedBufferParameteri64v_MACRO
-#define glGetNamedBufferParameteri64v_MACRO glGetNamedBufferParameteri64v
+#ifndef glGetNamedBufferParameteri64v_MACRO
+#define glGetNamedBufferParameteri64v_MACRO glGetNamedBufferParameteri64v
 #endif

```

Comment:
all these we don't need, actually I don't think we need the `COpenGLStateManagerImpl` and the `COpenGLStateManager` at all